pcdsdevices/areadetector/detectors.py

```diff
@@ -24,27 +25,127 @@ class PCDSDetectorBase(DetectorBase):
     """
     Standard area detector with no plugins.
     """
-    cam = ADComponent(cam.CamBase, ":")
+    cam = ADComponent(cam.CamBase, '')
 
 
 class PCDSDetector(PCDSDetectorBase):
     """
-    Standard area detector with standard plugins.
+    Standard area detector including all (*) standard PCDS plugins.
+    Notable plugins:
+        IMAGE2: reduced rate image, used for camera viewer
+        Stats2: reduced rate stats
+
+    (*) Currently excludes all PVAccess plugins:
+        IMAGE1:Pva
+        IMAGE2:Pva:
+        THUMBNAIL:Pva
+
+    Default port chains
+    -------------------
+    The default configuration of ports is as follows::
 
-    Geared towards analyzing a beam spot.
+        CAM -> CC1
+        CAM -> CC2
+        CAM -> HDF51
+        CAM -> IMAGE1:CC
+        CAM -> IMAGE1:Proc
+        CAM -> IMAGE1:ROI
+        CAM -> IMAGE2:CC
+        CAM -> IMAGE2:Over
+        CAM -> IMAGE2:Proc
+        CAM -> IMAGE2:ROI -> IMAGE2
+        CAM -> JPEG1
+        CAM -> NetCDF1
+        CAM -> Over1
+        CAM -> Proc1
+        CAM -> ROI1 -> IMAGE1
+        CAM -> ROI1 -> Stats1
+        CAM -> ROI2
+        CAM -> ROI3
+        CAM -> ROI4
+        CAM -> Stats2
+        CAM -> Stats3
+        CAM -> Stats4
+        CAM -> Stats5
+        CAM -> THUMBNAIL:CC
+        CAM -> THUMBNAIL:Over
+        CAM -> THUMBNAIL:Proc
+        CAM -> THUMBNAIL:ROI -> THUMBNAIL
+        CAM -> TIFF1
+        CAM -> Trans1
 
-    IMAGE2: reduced rate image
-    Stats2: reduced rate stats
+    Note
+    ----
+    Subclasses should replace 'cam' with that of the respective detector, such
+    as `ophyd.areadetector.cam.PilatusDetectorCam` for the Pilatus detector.
     """
-    image = Cpt(ImagePlugin, ':IMAGE2:', read_attrs=['array_data'])
```

Comment:
Not at all concerned about back-compat with colons. This however might cause issues as I think we do have code out there that refers to `my_det.image` and `my_det.stats`. We should drop these names in the long run as they are too vague, but do you think you could alias these maybe we `DeprecationWarning` until then? Thoughts?

Reply:
I like the numbers, this way the names match the screens as well. I was imagining a `property` based system.

Reply:
See latest changes, noting that the default warning is `UserWarning`.